e2e/cypress/support/api_commands.js

```diff
@@ -375,6 +375,26 @@ Cypress.Commands.add('apiEnablePluginById', (pluginId) => {
     });
 });
 
+/**
+ * Remove plugin by id.
+ *
+ * @param {String} pluginId - Id of the plugin to remove
+ * @param {Boolean} errorOnFailure - whether to fail if removing is unsuccessful
+ */
+Cypress.Commands.add('apiRemovePluginById', (pluginId, errorOnFailure) => {
```

Comment:
We have an existing command at https://github.com/mattermost/mattermost-webapp/pull/6361/files#diff-a8b7db926e1b4203e30b8e8265144cfbR330. Would that work as is? If not, maybe update?

Reply:
@saturninoabril the link you've posted in your comment seems wrong. What's the name of the existing command?

Reply:
O sorry about that. It's on L330 of this same file.
```
Cypress.Commands.add('apiRemovePluginById', (pluginId) => {
```